### PR TITLE
Ackquire AIK during attestation

### PR DESCRIPTION
**Pull request by arturkow2000**

### CODE REVIEW

File: src/client/crypto.rs [hunk 1]
```diff
@@ -81,9 +21,24 @@ impl RsaKey<'_> {
             }
         }
     }
+
+    pub fn load2(n: &[u8], e: &[u8]) -> Result<Self, ()> {
```
Comment (krystian-hebel) [inline src/client/crypto.rs]: Is it possible to store `e` as a slice so `load` can be used to load key from file, too?

File: src/client/signing.rs [hunk 2]
```diff
@@ -105,3 +57,23 @@ where
...
+/// Computes SHA-256 hash of inner object
+pub fn hash_signed_object<T>(trussed: &mut T, data: &[u8]) -> Result<trussed::types::ShortData, ()>
+where
+    T: trussed::client::CryptoClient,
+{
+    let signed_object = trussed::cbor_deserialize::<SignedObject>(data).map_err(|e| {
+        error!("Failed to deserialize signed object (outer): {}", e);
+    })?;
+
+    let sha = trussed::try_syscall!(trussed.hash(
```
Comment (krystian-hebel) [inline src/client/signing.rs:76]: Is this the only place where `MAX_MESSAGE_LENGTH` limits us?

It is sad that Trussed still uses `update` under the hood (https://github.com/trussed-dev/trussed/blob/main/src/mechanisms/sha256.rs#L43) but doesn't expose it. It should be possible to implement in a stateless way, by passing internal state (same size as hash, plus maybe additional integer for keeping size info) for each function...

Comment (arturkow2000) [reply src/client/signing.rs:76]: @krystian-hebel It limits file size too

File: src/client/crypto.rs (hunk 1 above)
Comment (arturkow2000) [reply src/client/crypto.rs]: @krystian-hebel You mean to store as u32? Yes, it is possible.

Comment (arturkow2000) [reply src/client/crypto.rs]: Fixed

File: src/client/signing.rs (hunk 2 above)
Comment (arturkow2000) [reply src/client/signing.rs:76]: Increased limit to 8192 and tested whether still works on nRF.

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**